steps/identity/appellant-contact-details/answer.html

```diff
@@ -1,34 +1,32 @@
-
+<div class="govuk-summary-list">
+    <dl class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">{{ content.cya.address.question | safe }}</dt>
+        <dd class="govuk-summary-list__value">
+            {{ fields.addressLine1.value | safe }}<br>
```

Comment:
This looks like a potential XSS vulnerability where HTML can be injected into the page,
has this been sanitised elsewhere?

What's the reason for this change?

Reply:
You mean adding the ` | safe` filter?

Reply:
Yes safe allows adding arbitrary html and should never be used on unsanitised user input, and should really just not be used on user input in general

Reply:
You're right, I had to add it to thousands of fields for Welsh and couldn't really check where every single one of them came from.
I guess I'll have to do a further scan to find them all.